test/import_order_timeout.lean

```diff
@@ -14,7 +14,7 @@ variables [add_comm_group N] [module R N]
 open function
 
 lemma injective_iff (f : M →ₗ[R] N) : function.injective f ↔ ∀ m, f m = 0 → m = 0 :=
-add_monoid_hom.injective_iff f.to_add_monoid_hom
+injective_iff_map_eq_zero f.to_add_monoid_hom
```

Comment:
I changed this line of `test/import_order_timeout` because it came up with grep. Let me know if that was bad.